tools/RAiDER/models/hres.py

```diff
@@ -24,14 +32,12 @@ def __init__(self):
         self._k3 = 3.75e3  # [K^2/Pa]
 
         # 9 km horizontal grid spacing. This is only used for extending the download-buffer, i.e. not in subsequent processing.
-        self._lon_res = 9. / 111
-        self._lat_res = 9. / 111
-        self._x_res = 9. / 111
-        self._y_res = 9. / 111
+        self._lon_res = 0.08108115
+        self._lat_res = 0.08108115
+        self._x_res = 0.08108115
+        self._y_res = 0.08108115
```

Comment:
Does this impact us when we have already download all models with 9/111?

Reply:
That is a good point; yes I suspect it will. We can wait on implementing this part until after done processing.

Reply:
Just a note that the float number is the value I get as the native grid resolution, which is very slightly different than the division result (0.0810810810...).